sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/literals.scala

```diff
@@ -62,7 +62,11 @@ object IntegerLiteral {
   }
 }
 
-case class Literal(value: Any, dataType: DataType) extends LeafExpression {
+case class Literal(var value: Any, dataType: DataType) extends LeafExpression {
```

Comment:
we should create a object Literal and add an apply function that converts string into UTF8String, rather than making this a var.